### PR TITLE
#23 Option to pass custom headers

### PR DESCRIPTION
**Pull request by reda-alaoui**
https://github.com/observing/thor/issues/23

### CODE REVIEW

**Comment by reda-alaoui** [comment]
Considering this project as dead.

**Comment by 3rd-Eden** [comment]
Cool.

**Comment by reda-alaoui** [comment]
No activity on PRs for one year. 
I say that the project is dead, and then magically someone appears and start to process them :)

Sorry if I looked agressive, this wasn't my intent. I was juste cleaning my open PRs. If you are interested in those PRs I can reopen them.

I know that maintaining a project is hard ;)

File: bin/thor [hunk 1]
```diff
@@ -47,6 +62,21 @@ var cluster = require('cluster')
   , received = 0
   , robin = [];
 
+//
+// Process headers
+//
+var headers = {};
+cli.header.forEach(function(header){
+  var keyValue = header.split(':');
```
Comment (robertoronderosjr) [inline bin/thor:70]: Header Val might have ':' therefore the split will be wrong. Try:
var keyValue = header.split(/:/);
var key = keyValue.shift();
var value = keyValue.join(':');

**Review by robertoronderosjr — CHANGES_REQUESTED** [review]

**Pull request closed without merging**